src/pytorch_lightning/utilities/memory.py

```diff
@@ -96,38 +96,6 @@ def garbage_collection_cuda() -> None:
             raise
 
 
-def get_gpu_memory_map() -> Dict[str, float]:
```

Comment:
Good find @akihironitta! 
Do you think we should add it to the `src/pytorch_lightning/_graveyard`? cc @carmocca

Reply:
This is something @Borda found actually :)

I hope I did this right in 13afae3 as this is my first time burying something in the graveyard :)